CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing
+
+Thank you for your interest in improving InsightConnect! Please review our [Code of Conduct] before making contributions.
+
+There are multiple ways to contribute beyond writing code. These include:
+
+- [Submit bugs and feature requests] with detailed information about your issue or idea.
+- [Help fellow users with open issues] or [help fellow committers test recent pull requests].
+- [Report a security vulnerability in InsightConnect's plugins] to Rapid7.
+- Submit an updated or brand new plugin!  We are always eager for new
+  integrations or features. Don't know where to start? Check out the [developer documentation].
+
+Here is a short list of do's and don'ts to make sure *your* valuable contributions actually make
+it into production.  If you do not care to follow these rules, your contribution
```

Comment:
```suggestion
it into production.  If you do not care to follow these rules, your contribution will be rejected
```
Using `closed` implies a pull request is used. Might want to play around with the wording.